rclcpp/include/rclcpp/exceptions.hpp

```diff
@@ -198,6 +198,13 @@ class InvalidParameterValueException : public std::runtime_error
   using std::runtime_error::runtime_error;
 };
 
+/// Thrown if passed parameter description is invalid.
+class InvalidParameterDescriptionException : public std::runtime_error
+{
+  // Inherit constructors from runtime_error.
+  using std::runtime_error::runtime_error;
+};
+
```

Comment:
Is this used somewhere? 

- Any combination of `to` / `from` and `step` are basically valid.
- The name and type of a given descriptor are ignored, so it matter if they don't match the name and type of the associated parameter.
- Having only an integer range, only a floating point range, both at the same time or none shouldn't cause trouble.

Is there something I'm missing?

Reply:
Forgot to delete it. I was going to use it, but I finally didn't.